tests/console/firewalld.pm

```diff
@@ -0,0 +1,142 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test FirewallD basic usage
+# Maintainer: Alexandre Makoto Tanno <atanno@suse.com>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+
+sub run {
+    select_console("root-console");
+
+    # Check Service State, enable it if necessary
+    record_info 'Check Service State';
+    #assert_script_run("if [ `firewall-cmd --state > /dev/null 2>&1 ; echo $?` -eq 252 ]; then systemctl start firewalld; fi");
+    assert_script_run("if [ `systemctl is-active firewalld > /dev/null 2>&1 ; echo $?` -eq 0 ]; then systemctl start firewalld; fi");
+
+    # Test #1 - Stop firewalld then start it
+    record_info 'Test #1', 'Test: Stop firewalld, then start it';
+    assert_script_run("systemctl stop firewalld");
```

Comment:
there is wrapper function in openQA for systemctl call :
```suggestion
    systemctl "stop firewalld";
```

Reply:
systemctl("stop firewalld");